packages/loot-core/src/client/reducers/modals.ts

```diff
@@ -10,6 +10,16 @@ const initialState: ModalsState = {
 export function update(state = initialState, action: Action): ModalsState {
   switch (action.type) {
     case constants.PUSH_MODAL:
+      // special case: don't show the keyboard shortcuts modal if there's already a modal open
+      if (
+        action.modal.name.endsWith('keyboard-shortcuts') &&
+        (state.modalStack.length > 0 ||
+          window.document.querySelector(
+            'div[data-testid="filters-menu-tooltip"]',
```

Comment:
We should check this before the pushModal call in the desktop-client package and not in loot-core

Reply:
We can't. As far as I can tell.  I did not see any way to get the `modalStack` from that particular file.

Reply:
Sorry, what I meant is the window object check

Reply:
I guess that could move, but then the logic for not showing a modal is split across two places.  It is already bad enough having it exist at all, I personally think having it in one place would be the better approach.

Reply:
(And I would strongly prefer that "one place" be before the pushModal call!)